test_scripts/API/AlertManeuver/009_ATF_AlertManeuver_Navi_AlertManeuver_WARNINGS.lua

```diff
@@ -0,0 +1,157 @@
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"AlertManeuver")
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+local resultCodes = {"SUCCESS", "WARNINGS", "WRONG_LANGUAGE", "RETRY", "SAVED"}
+
+for i=1,#resultCodes do
+  Test["TestStep__AlertManeuver_Navi_AlertManeuver_WARNINGS_TTS_Apeak_"..resultCodes[i]] = function(self)
+    local cor_id_alertmaneuver = self.mobileSession:SendRPC("AlertManeuver",
+    {ttsChunks =
+        {{
+          text ="FirstAlert",
+          type ="TEXT",
+        },
+        {
+          text ="SecondAlert",
+          type ="TEXT",
+      }},
+      softButtons =
+        {{
+          type = "BOTH",
+          text = "Close",
+          image =
+          {
+            value = "icon.png",
+            imageType = "DYNAMIC",
+          },
+          isHighlighted = true,
+          softButtonID = 821,
+          systemAction = "DEFAULT_ACTION",
+      }}
+    })
+
+    EXPECT_HMICALL("Navigation.AlertManeuver",
+    {
+      softButtons =
+        {{
+          type = "BOTH",
+          text = "Close",
+          image = { imageType = "DYNAMIC"},
+          isHighlighted = true,
+          softButtonID = 821,
+          systemAction = "DEFAULT_ACTION",
+      }},
+      appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+    })
+    :ValidIf(function(_,data)
+      local value_Icon = storagePath .. "icon.png"
+      if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
```

Comment:
@vvvakulenko, image is part of softButton array, you should use: data.params.softButtons[1].image.value to verify value of image.